visualisation.md

```diff
@@ -1,5 +1,59 @@
 ---
-title: Beneficial Ownership Visual Language
```

Comment:
Terminology wise - I'd prefer 'Visualisation language for ownership networks' (or similar if better acronym can be wrought) rather than BOVS right now.

For something we would call a standard we would want to go through more of a governance process - and I think better to sidestep all the baggage people bring when there is talk of standards.

Reply:
Fair enough. I think a name beginning "BO" is really clear and also positions it as something we want to be generally used. I started with "BOVL" (Visualisation Language) but that is not a nice acronym to pronounce! "BOVS" is easier to say, like BODS. How about BO Visualisation System? Then if it ever becomes a "Standard", we don't need to change the acronym.

Reply:
I'm good with 'Visualisation System'

Added bonus - we can ask of diagrams: "Is it BOVered?"

Reply:
XD